svenfire99/project_bioscooop
# Caterer CRUD

Made the caterer side complete you can now add and remove food items. To test my pull request commit your changes by doing the following in the terminal:

- git status to see which files you have modified
- git add .
- git commit -m "your message about what you have done"
- git push origin HEAD
- git checkout Ahmet

How to test my work:
When you run the code press L in the build section after that log in with the following username and password (caterer) they are both the same. When you are logged in try to add an food item it will first ask for a name of the food item after that it will ask a price please use an **PLEASE USE AM INT NOT A FLOAT IF YOU USE AN FLOAT IT WILL NOT BE ACCEPTED THIS WILL BE FIXED IN THE FEATURE IF NEEDED**.You can also use the remove function you will see an list of the fooditems with the price and ID select the ID in bracets. If that all works then you can switch back to your own branch. to do that you just type in the terminal the following code.
- git checkout yourownbranchname

PS: if you have any questions please make sure to contact me.
